web/templates/cluster_hana.html.tmpl

```diff
@@ -0,0 +1,115 @@
+                            <span class="text-muted">{{ $HANAHealthState }}</span>
+                        {{- else  if or (eq $HANAHealthState "2") (eq $HANAHealthState "3")  }}
+                            <i class="eos-icons eos-18 text-warning">fiber_manual_record</i>
+                            <span class="text-muted">{{ $firstNode.HealthState }}</span>
+                        {{- else  if or (eq $HANAHealthState "1")  }}
+                            <i class="eos-icons eos-18 text-danger">fiber_manual_record</i>
+                            <span class="text-muted">{{ $HANAHealthState }}</span>
+                        {{- else }}
+                            -
+                        {{- end}}
+                    </div>
+                    <div class="col-6 mt-5">
+                        <strong>HANA secondary sync state:</strong><br>
+                        {{ $secondarySyncState := .Nodes.HANASecondarySyncState }}
+                        {{ $badgeClass := "badge-primary" }}
+                        {{- if eq $secondarySyncState "SFAIL" }}
+                            {{ $badgeClass =  "badge-danger" }}
+                        {{- end }}
+                        <span class="badge badge-pill {{ $badgeClass }} ml-0">{{ $secondarySyncState }}</span>
+                    </div>
+
+                    <div class="col-3 mt-5">
+                        <strong>Stonith type:</strong><br>
+                        <span class="text-muted">{{ .Cluster.FencingType }}</span>
+                    </div>
+                    <div class="col-3 mt-5">
+                        <strong>CIB last written:</strong><br>
+                        <span class="text-muted">{{ .Cluster.Crmmon.Summary.LastChange.Time }}</span>
+                    </div>
+                    <div class="col-6 mt-5">
+                        <strong>HANA system replication operation mode:</strong><br>
+                        <span class="text-muted">{{ index $firstNode.Attributes "hana_prd_op_mode" }}</span>
+                    </div>
+                </div>
+            </div>
+            <div class="col-sm-3">
+                <div class="mt-3">
+                    {{ template "health_container" .HealthContainer }}
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <h4>Stopped resources</h4>
+    <div class="row mt-4 mb-4">
+        <div class="col-xl-12">
+            {{- range .StoppedResources }}
+                <span class="badge badge-pill badge-secondary ml-0">{{ .Id }}</span>
+            {{- else }}
+                <p class="text-muted">No stopped resources</p>
+            {{- end}}
+        </div>
+    </div>
+
+    <h3>Pacemaker Site details</h3>
+    <div class="row mt-4">
+        <div class="col-xl-12">
+            {{ template "sites" .Nodes.GroupBySite }}
+        </div>
+    </div>
+    <hr>
+
+    <h3>SBD/Stonith</h3>
+    {{- if .Cluster.IsFencingSBD }}
+        {{ template "sbd" .Cluster.SBD }}
+        <hr/>
```

Comment:
Not sure if the <hr> closure is needed, but in any case, is this the right place?

Reply:
It took me a while to understand but then I realized github was rendering the html ahah

No, you are right it is not needed I will remove this

Reply:
haha, fixed my comment by adding the `<hr>` as code